logging/config/cluster-logging-curator.adoc

```diff
@@ -5,14 +5,11 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-In {product-title} {product-version}, log curation is performed by Elasticsearch Curator, based on a xref:../../logging/config/cluster-logging-log-store.html#cluster-logging-elasticsearch-retention_cluster-logging-store[retention policy] that you define.
+In {product-title} {product-version}, log curation is performed by ElasticSearch `IndexManagement` - the `elasticsearch-im-app`, `elasticsearch-im-audit` and `elasticsearch-im-infra` cronjobs, which you can configure using the xref:../../logging/config/cluster-logging-log-store.html#cluster-logging-elasticsearch-retention_cluster-logging-store[retentionPolicy].
 
-The Elasticsearch Curator tool removes Elasticsearch indices that use the data model prior to {product-title} 4.5. You can modify the Curator index retention policy for your old data.
+Elasticsearch Curator will be deprecated in a future version. If the cluster was deployed on {product-title} version 4.5 or later, Elasticsearch Curator is not required.
```

Comment:
Shouldnt this say "is deprecated and removed in a future version"?  It will actually no longer be available as soon as v5.1

Reply:
@jcantrill Curator was never deprecated:
https://coreos.slack.com/archives/GGUR75P60/p1618449764473900

Reply:
Updated content.